src/components/Form/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+import { Field, ErrorMessage, useFormikContext } from 'formik';
+import PropTypes from "prop-types";
+
+
+export const FocusError = () => {
+  const { errors, isSubmitting, isValidating } = useFormikContext();
+
+  useEffect(() => {
+    if (isSubmitting && !isValidating) {
+      let keys = Object.keys(errors);
+      if (keys.length > 0) {
+        const selector = `[name=${keys[0]}]`;
+        const errorElement = document.querySelector(selector);
+        if (errorElement) {
+          errorElement.focus();
+        }
+      }
+    }
+  }, [errors, isSubmitting, isValidating]);
+
+  return null;
+};
+
+
+export const FormGroup = (props) => {
+  const { children, component, errors, helpText, label, name, required, rows, touched, type } = props;
+  return (
+    <div className="form-group">
+      { type !== "checkbox" && <label htmlFor={name}>{label}</label> }
+      <Field
+        tabIndex="0"
+        className={ errors && errors[name] && touched[name] ? "is-invalid" : null }
+        type={type}
+        name={name}
+        id={name}
+        component={component}
+        rows={rows}
+        children={children}
+        aria-invalid={errors && errors[name] && touched[name] ? 'true' : null}
+        aria-describedby={errors && errors[name] && touched[name] ? `${name}-error` : null}
```

Comment:
Can we add `desc-${name}` as an initial aria-describedby value that corresponds to the `helpText`, and then when there is an error, add this additional `${name}-error` value? I couldn't figure out how to get both.

Reply:
I think that would look something like:
```
{errors && errors[name] && touched[name] ? `${name}-error desc-${name}` : desc-${name}}
```
The potential issue I see here is that there isn't always help text, so this may in some cases point to a non-existent DOM element.